src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidatorTest.java

```diff
@@ -0,0 +1,178 @@
+
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.TRUST_NAME_FIELD);
+        String validationMessage = ValidationMessages.INVALID_CHARACTERS_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenCreationDateFieldIsNull() {
+        trustDataDtoList.get(0).setCreationDate(null);
+        Errors errors = trustDetailsValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CREATION_DATE_FIELD);
+        String validationMessage = ValidationMessages.NOT_NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenCreationDateFieldIsInThePast() {
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(1970,1, 1));
+        Errors errors = trustDetailsValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testErrorReportedWhenCreationDateIsInTheFuture() {
+        trustDataDtoList.get(0).setCreationDate(LocalDate.now().plusDays(1));
+        Errors errors = trustDetailsValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT);
+
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CREATION_DATE_FIELD);
+        String validationMessage = ValidationMessages.DATE_NOT_IN_PAST_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenUnableToObtainAllTrustInfoFieldIsNull() {
+        trustDataDtoList.get(0).setUnableToObtainAllTrustInfo(null);
+        Errors errors = trustDetailsValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT);
+
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.UNABLE_TO_OBTAIN_ALL_TRUST_INFO_FIELD);
+        String validationMessage = ValidationMessages.NOT_NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenUnableToObtainAllTrustInfoFieldNotEmpty() {
+        trustDataDtoList.get(0).setUnableToObtainAllTrustInfo(false);
+        Errors errors = trustDetailsValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT);
+
+        assertFalse(errors.hasErrors());
+    }
+
+    private void assertError(String qualifiedFieldName, String message, Errors errors) {
+        Err err = Err.invalidBodyBuilderWithLocation(qualifiedFieldName).withError(message).build();
+        assertTrue(errors.containsError(err));
+    }
+
```

Comment:
Blank lines here could be removed.

Reply:
thanks @mwestacott I have removed the empty lines.